data/json/npcs/exodii/exodii_weaponmaster_talk.json

```diff
@@ -0,0 +1,76 @@
+[
+  {
+    "id": "TALK_WEAPONMASTER_INTRO_1",
+    "type": "talk_topic",
+    "//": "mute by choise, and deaf most of the time (don't like to turn on the hearing cbm unless for job)",
+    "dynamic_line": "&What you see looks like the back of a mechanical spider, with at least a dozen crude but polished metal arms going from the enormously wide spine and shoulders.  Spine of a creature, meanwhile, sits in an awkward-looking chair, doing something on a wide, heavily lighted workbench.  Creature seems to be busy with something.",
+    "responses": [
+      { "text": "&Wait a bit.", "effect": "start_trade", "topic": "TALK_WEAPONMASTER_INTRO_2A" },
+      { "text": "Hello?", "topic": "TALK_WEAPONMASTER_INTRO_2B" }
+    ]
+  },
+  {
+    "id": "TALK_WEAPONMASTER_INTRO_2A",
+    "type": "talk_topic",
+    "dynamic_line": "&It doesn't react at your presence at all.",
+    "responses": [
+      { "text": "&Wait in silence.", "effect": "start_trade", "topic": "TALK_WEAPONMASTER_INTRO_3" },
+      { "text": "Hello!?", "topic": "TALK_WEAPONMASTER_INTRO_3" }
+    ]
+  },
+  {
+    "id": "TALK_WEAPONMASTER_INTRO_2B",
+    "type": "talk_topic",
+    "dynamic_line": "&It seems to not hear you at all.",
+    "responses": [
+      { "text": "&Wait for longer.", "effect": "start_trade", "topic": "TALK_WEAPONMASTER_INTRO_3" },
+      { "text": "Hello?", "topic": "TALK_WEAPONMASTER_INTRO_3" }
+    ]
+  },
+  {
+    "id": "TALK_WEAPONMASTER_INTRO_3",
+    "type": "talk_topic",
+    "dynamic_line": "&A few seconds later, they seem to finish whatever they were busy with and slowly turn their chair to you.  Their front looks no less exotic than their back; the abundance of metal arms looks even more extraordinary next to the clear lack of any legs.  What you initially thought to be a chair is simply a rotating platform, sturdy enough to support the weight of the entire body of a creature.  Few cameras, that seems to be a replacement for eyes, look at you motionlessly, while the forehead of the creature has the surprisingly familiar sign \"IX\" painted on.",
+    "responses": [ { "text": "Who-…  Wha-…  What are you?", "topic": "TALK_WEAPONMASTER_INTRO_4" } ]
+  },
+  {
+    "id": "TALK_WEAPONMASTER_INTRO_4",
+    "type": "talk_topic",
+    "dynamic_line": "&Still completely mute, it shows you something you completely missed: The entire wall behind the workbench was full of various weapons: some as big as modern rifles, a few as large as machineguns installed on aircraft, and one that looked like dismantled tank turrets.  Melee weapons were not left behind, from moderate-sized swords and clubs to warhammers twice the human height and what you can describe only as \"boots for mecha\".  Then it pointed to the sign near you that showed the carved pictograms of coins, some unknown for you sign, and then a bullet - all things considered, it is either a weaponmaster or a weapon merchant.",
```

Comment:
This whole thing needs to be restructured

Still completely mute, it shows you something you had initially missed. The entire wall behind the workbench is full of various weapons; some as big as modern rifles, a few as large as the machine guns installed on aircraft, and one that looks like a dismantled tank turret.  Melee weapons were not forgotten, from moderately-sized swords and clubs to war hammers twice the height of a human, and what you can describe only as \"boots for mecha\".  Then it points to a sign near you that shows the carved pictograms of coins, some unknown to you, next to a pictogram of a bullet - all things considered, this creature is either a weapons master or a weapons merchant.